cmd/configure_test.go

```diff
@@ -32,11 +32,11 @@ func TestConfigure(t *testing.T) {
 		testCase{
 			desc:           "It overwrites the flags that are passed, without losing the ones that are not.",
 			args:           []string{"fakeapp", "configure", "--token", "c", "--skip-auth"},
-			existingUsrCfg: &config.UserConfig{Token: "token-c", Workspace: "/workspace-c"},
-			expectedUsrCfg: &config.UserConfig{Token: "c", Workspace: "/workspace-c"},
+			existingUsrCfg: &config.UserConfig{Token: "token-c", Workspace: "/workspace-c", APIBaseURL: "http://example.com"},
```

Comment:
For a second I thought the APIBaseURL was some how being obtained from a previous testcase using `http://example.com`. Maybe having a workspace and APIBaseURL that is different from other test cases or something like `Workspace: "/unmodified-workspace-path", APIBaseURL: "http://unmodified.example.com/"` would help illustrate the point. Or a description like "Unspecified flags do not overwrite previously configured settings"

Reply:
Nice, yeah, that would be much clearer.

Reply:
I went ahead and did a complete overhaul of the test data values in that test to help clarify intention throughout. I like it a lot better! Thanks for the suggestion.

Reply:
Looks good!